src/zmq/zmqabstractnotifier.h

```diff
@@ -5,12 +5,13 @@
 #ifndef BITCOIN_ZMQ_ZMQABSTRACTNOTIFIER_H
 #define BITCOIN_ZMQ_ZMQABSTRACTNOTIFIER_H
 
-#include <zmq/zmqconfig.h>
+#include <memory>
 
```

Comment:
Missing a bunch of includes here. See appveyor CI result.

Reply:
I missed that comment - I've now revised the commit to include `<string>`.  There are still CI errors, though, which I don't understand so far.  Has anyone an insight of where they come from?  I'll try to resolve them now.